packages/cozy-client/src/store/index.js

```diff
@@ -49,33 +49,65 @@ export class StoreProxy {
 
 const initialState = { documents: {}, queries: {} }
 
-const combinedReducer = (state = initialState, action) => {
+const combinedReducer = performanceApi => (state = initialState, action) => {
+  const markName = performanceApi.mark('combinedReducer')
+
   if (action.type == RESET_ACTION_TYPE) {
+    performanceApi.measure({
+      markName: markName,
+      measureName: `${markName} reset`,
+      category: 'CozyClientStore'
+    })
     return initialState
   }
   if (!isQueryAction(action) && !isMutationAction(action)) {
+    performanceApi.measure({
+      markName: markName,
+      measureName: `${markName} query or mutate`,
+      category: 'CozyClientStore'
+    })
     return state
   }
   if (action.update) {
     const proxy = new StoreProxy(state)
     action.update(proxy, action.response)
-    return {
+    const newState = {
       documents: proxy.getState().documents,
       queries: queries(
+        performanceApi,
         proxy.getState().queries,
         action,
         proxy.getState().documents
       )
     }
+    performanceApi.measure({
+      markName: markName,
+      measureName: `${markName} update`,
+      category: 'CozyClientStore'
+    })
+    return newState
   }
 
   const nextDocuments = documents(state.documents, action)
   const haveDocumentsChanged = nextDocuments !== state.documents
 
-  return {
+  const queriesResult = {
     documents: nextDocuments,
-    queries: queries(state.queries, action, nextDocuments, haveDocumentsChanged)
+    queries: queries(
+      performanceApi,
+      state.queries,
+      action,
+      nextDocuments,
+      haveDocumentsChanged
+    )
   }
+
+  performanceApi.measure({
+    markName: markName,
+    measureName: `${markName} default`,
+    category: 'CozyClientStore'
+  })
```

Comment:
Fixed [here](https://github.com/cozy/cozy-client/compare/ee1b801325c7d7db3ca12d34712541c1b9b95165..fabe643260d358b4cd558200d45dec15387e476b)

I renamed it `${markName} queries` but I'm wondering if this is correct too? Those can be queries or mutation? Does "queries" implies this can be also mutation?